packages/fury-adapter-oas3-parser/test/integration/components-test.js

```diff
@@ -4,14 +4,21 @@ const testParseFixture = require('./testParseFixture');
 const fixtures = path.join(__dirname, 'fixtures', 'components');
 
 describe('components', () => {
-  it("'Path Item Object' parameter references", () => {
-    const file = path.join(fixtures, 'path-item-object-parameters');
-    return testParseFixture(file);
-  });
-
-  it("'Path Item Object' parameter referencing unsupported parameter", () => {
-    const file = path.join(fixtures, 'path-item-object-parameters-unsupported-parameter');
-    return testParseFixture(file);
+  describe('Path Item Object', () => {
+    it('handles parameter references', () => {
+      const file = path.join(fixtures, 'path-item-object-parameters');
+      return testParseFixture(file);
+    });
+
+    it('handles multiple references to same parameter', () => {
```

Comment:
Using an integration test because this particular bug is with interplay of using parameters referenced from multiple different components with element `freeze()`. There's a lot of setup state for this and it's simpler to test referencing in this way.